app/core/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -313,6 +313,18 @@ public boolean create() throws MessagingException {
         return true;
     }
 
+    public static void createLocalFolder(LocalStore localStore, String internalId, String folderName)
```

Comment:
Why not make this an instance method on `LocalStore`?

Reply:
done